src/views/GovernanceTaskList/index.tsx

```diff
@@ -308,6 +309,17 @@ const GovernanceTaskList = () => {
                         <br />
                       </>
                     )}
+                  <UswdsReactLink
+                    className={`display-inline-block ${
+                      status === 'READY_FOR_GRT' ? 'usa-button' : ''
+                    }`}
+                    target="_blank"
+                    variant="unstyled"
+                    data-testid="prepare-for-grt-cta"
+                    to="/help/it-governance/prepare-for-grt"
+                  >
+                    Prepare for the GRT meeting (opens in new tab)
+                  </UswdsReactLink>
                   <BusinessCaseDraftCta systemIntake={systemIntake} />
```

Comment:
These two component render inline instead of blocked

![Screen Shot 2022-05-23 at 2 17 36 PM](https://user-images.githubusercontent.com/95709965/169882080-84202e6c-72fd-4207-ba54-77eae389a3c7.png)